nodeup/pkg/model/kube_apiserver.go

```diff
@@ -64,6 +64,16 @@ func (b *KubeAPIServerBuilder) Build(c *fi.ModelBuilderContext) error {
 		kubeAPIServer = *b.NodeupConfig.APIServerConfig.KubeAPIServer
 	}
 
+	if b.UsesSecondaryIP() {
```

Comment:
This means we'll start setting --advertise-address for AWS clusters as well, correct? Do we need that set? If so could we do that in a separate PR?

Reply:
It is needed, otherwise the advertised address remains the external IP, which is firewalled.
Changed the condition to `b.CloudProvider == kops.CloudProviderHetzner` so it only affects Hetzner for now.